vpr/src/route/route_tree_timing.cpp

```diff
@@ -316,6 +331,7 @@ add_subtree_to_route_tree(t_heap* hptr, t_rt_node** sink_rt_node_ptr) {
 
         rt_node->u.child_list = linked_rt_edge;
         rt_node->inode = inode;
+        rt_node->ipin = OPEN;
```

Comment:
Explain why:  not valid for non-SINK nodes.

Reply:
Done.